nifi-toolkit/nifi-toolkit-encrypt-config/src/main/groovy/org/apache/nifi/properties/ConfigEncryptionTool.groovy

```diff
@@ -1091,7 +1095,7 @@ class ConfigEncryptionTool {
             encryptedProperties.setProperty(key, plainProperties.getProperty(key))
         }
         NiFiProperties mergedProperties = new StandardNiFiProperties(encryptedProperties)
-        logger.info("Final result: ${mergedProperties.size()} keys including ${ProtectedNiFiProperties.countProtectedProperties(mergedProperties)} protected keys")
+        logger.info("Final result: ${mergedProperties.size()} keys including ${ProtectedNiFiProperties.countProtectedProperties(mergedProperties, keyHex)} protected keys")
```

Comment:
I think the implementation of `ProtectedNiFiProperties#countProtectedProperties()` needs to change so that it doesn't require the key hex or ID in order to count the protected properties.

Reply:
I removed the unnecessary key parameter; the method passes an empty string along to the ctor, which produces a null SPP, which provides the count as expected.